docs: update 04-30-10-gcp-nfs-volume.md based on latest changes (#519)
kyma-project/cloud-manager: docs/user/resources/04-30-10-gcp-nfs-volume.md

Manually create a non-default IpRange with specified Classless Inter-Domain Routing (CIDR) and use it only in advanced cases of network topology
when you want to be in control of the network segments in order to avoid range conflicts with other networks.

> You must specify the GcpNfsVolume capacity. Scalability and minimum/maximum capacity depends on the chosen GCP Filestore tier.

Reviewer: ```suggestion
You must specify the GcpNfsVolume capacity. Scalability and minimum/maximum capacity depend on the chosen GCP Filestore tier.
```

| **ipRange**                 | object              | Optional IpRange reference. If omitted, default IpRange will be used, if default IpRange does not exist, it will be created.                                                                                                                                                                                                                                                                                            |
| **ipRange.name**            | string              | Name of the existing IpRange to use.                                                                                                                                                                                                                                                                                                                                                                                    |

> | **capacityGb**              | int                 | Capacity of the provisioned volume in GiB. For a given tier range, a valid capacity not only needs to be between the minimum and maximum inclusive, but its difference from min needs to be divisible to scalability. For example 1200 is not a valid capacity for Zonal tier but 1280 (1024 + 256) is valid. To learn more, read [GCP Filestore service tiers](https://cloud.google.com/filestore/docs/service-tiers). |

Reviewer: ```suggestion
| **capacityGb**              | int                 | Capacity of the provisioned volume in GiB. For a given tier range, a valid capacity must be between the minimum and maximum inclusive, and its difference from min must be divisible to scalability. For example, 1200 is not a valid capacity for the Zonal tier but 1280 (1024 + 256) is valid. To learn more, read [GCP Filestore service tiers](https://cloud.google.com/filestore/docs/service-tiers). |
```
